src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -28,7 +28,11 @@ public class RpcInitializationService : IManagedHostedService
         {
             {
                 OSPlatform.Windows,
-                new List<string>() { RpcWorkerConstants.JavaLanguageWorkerName }
+                new List<string>()
+                {
+                    RpcWorkerConstants.JavaLanguageWorkerName,
```

Comment:
No need to update the list here. This is legacy before we had placeholder pools. I will remove this soon.

Reply:
Great, this is what I suspected. I will revert this change.

Reply:
No, wait. @pragnagopa, reverting this breaks the `RpcInitializationService_Initializes_RpcServerAndChannels_Windows_PlaceHolderMode test`. The following expectation is not met:
``` csharp
_mockLanguageWorkerChannelManager.Verify(m => m.InitializeChannelAsync(RpcWorkerConstants.PowerShellLanguageWorkerName), Times.Once);
```
Do you think this is a correct expectation?

Reply:
Like rest of the languages in the test, except of Java this needs to 
`mockLanguageWorkerChannelManager.Verify(m => m.InitializeChannelAsync(RpcWorkerConstants.PowerShellLanguageWorkerName), Times.Never);`